azext_iot/smoke_tests/commands_test.ps1

```diff
@@ -0,0 +1,109 @@
+$commands += "az iot device send-d2c-message -g $resource_group_name -n $iothub_name -d $device_id --data 'Test Message'"
+
+# IoT Edge
+$commands += "az iot edge set-modules -g $resource_group_name --hub-name $iothub_name --device-id $device_id --content $edge_module_content"
+$commands += "az iot edge deployment create -g $resource_group_name -d $edge_deployment_name -n $iothub_name --content $edge_deployment_content --target-condition $edge_deployment_condition --priority 10 --metrics $edge_deployment_metrics --layered"
+$commands += "az iot edge deployment show -g $resource_group_name -d $edge_deployment_name -n $iothub_name"
+
+# IoT DPS
+$commands += "az iot dps create -g $resource_group_name --name $dps_name"
+$commands += "az iot dps compute-device-key -g $resource_group_name --key $dps_endorsement_key --registration-id $dps_registration_id"
+$commands += "az iot dps enrollment-group create -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_group_id"
+$commands += "az iot dps enrollment-group show -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_group_id"
+$commands += "az iot dps enrollment create -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_id --attestation-type tpm --endorsement-key $dps_endorsement_key"
+$commands += "az iot dps enrollment show -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_id"
+$commands += "az iot dps registration list -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_group_id"
+$commands += "az iot dps connection-string show -g $resource_group_name --dps-name $dps_name --all"
+
+# Resource Cleanup
+$commands += "az iot hub module-identity delete -g $resource_group_name -m $hub_module_id -d $device_id -n $iothub_name"
+$commands += "az iot hub configuration delete -g $resource_group_name -c $hub_config_name -n $iothub_name"
+$commands += "az iot hub device-identity delete -g $resource_group_name -n $iothub_name -d $device_id"
+$commands += "az iot edge deployment delete -g $resource_group_name -d $edge_deployment_name -n $iothub_name"
+$commands += "az iot dps enrollment delete -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_id"
+$commands += "az iot dps enrollment-group delete -g $resource_group_name --dps-name $dps_name --enrollment-id $dps_enrollment_group_id"
+$commands += "az iot dps delete -g $resource_group_name --name $dps_name"
+$commands += "az iot hub delete -g $resource_group_name --name $iothub_name"
+
+Write-Host "`r`nRunning smoke test commands..."
+
+# Execute commands
+foreach ($command in $commands) {
+    try {
+        Invoke-Expression $command -OutVariable standardOut 2>&1 >$null
+    }
+    catch {
+        # Warning should not be considered a failure
+        if ($_ -notlike "*warning*" ) {
```

Comment:
Commands can be appended with the `--only-show-errors` flag to hide warnings.

Reply:
Thank you, will make the update (helps remove weird logic)